tests/build_/test_build_requestv2.py

```diff
@@ -0,0 +1,699 @@
+"""
+Copyright (c) 2018 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+import copy
+import glob
+import json
+import os
+import fnmatch
+from pkg_resources import parse_version
+import shutil
+import six
+import sys
+
+from osbs.build.build_requestv2 import BuildRequestV2
+from osbs.constants import DEFAULT_OUTER_TEMPLATE
+from osbs.exceptions import OsbsValidationException
+from osbs.repo_utils import RepoInfo, RepoConfiguration
+
+from flexmock import flexmock
+import pytest
+
+from tests.constants import (INPUTS_PATH, TEST_BUILD_CONFIG, TEST_BUILD_JSON,
+                             TEST_COMPONENT, TEST_GIT_BRANCH, TEST_GIT_REF,
+                             TEST_GIT_URI, TEST_GIT_URI_HUMAN_NAME,
+                             TEST_FILESYSTEM_KOJI_TASK_ID, TEST_SCRATCH_BUILD_NAME,
+                             TEST_ISOLATED_BUILD_NAME)
+# These are used as fixtures
+from tests.fake_api import openshift, osbs  # noqa
+
+USE_DEFAULT_TRIGGERS = object()
+
+
+class NoSuchPluginException(Exception):
+    pass
+
+
+def get_sample_prod_params():
+    return {
+        'git_uri': TEST_GIT_URI,
+        'git_ref': TEST_GIT_REF,
+        'git_branch': TEST_GIT_BRANCH,
+        'user': 'john-foo',
+        'component': TEST_COMPONENT,
+        'base_image': 'fedora:latest',
+        'name_label': 'fedora/resultingimage',
+        'koji_target': 'koji-target',
+        'sources_command': 'make',
+        'platforms': ['x86_64'],
+        'filesystem_koji_task_id': TEST_FILESYSTEM_KOJI_TASK_ID,
+        'build_from': 'image:buildroot:latest',
+    }
+
+
+def get_secret_mountpath_by_name(build_json, name):
+    secrets = build_json['spec']['strategy']['customStrategy']['secrets']
+    named_secrets = [secret for secret in secrets
+                     if secret['secretSource']['name'] == name]
+    assert len(named_secrets) == 1
+    secret = named_secrets[0]
+    assert 'mountPath' in secret
+    return secret['mountPath']
+
+
+class TestBuildRequestV2(object):
+    def test_inner_template(self):
+        br = BuildRequestV2('something')
+        with pytest.raises(RuntimeError):
+            assert br.inner_template
```

Comment:
`assert br.inner_template` there is slightly confusing, because at first glance it looks like the test thinks it should be truthy. Maybe just `br.inner_template`? Likewise below.

Reply:
Fixed.